# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
# Commit: a64e03b3e760
# Full Hash: a64e03b3e760528372218bc42f3c02f9447c5a38
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Handle call shutdown in MediaSessionConduit::GetCallStats() and callers. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D124375
# ==============================================================================

diff -r cc2d56c5c678 -r a64e03b3e760 dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h
--- a/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Thu Sep 02 12:19:23 2021 +0200
+++ b/dom/media/webrtc/libwebrtcglue/MediaConduitInterface.h	Thu Sep 02 13:55:29 2021 +0200
@@ -159,7 +159,7 @@
   virtual Maybe<RefPtr<AudioSessionConduit>> AsAudioSessionConduit() = 0;
   virtual Maybe<RefPtr<VideoSessionConduit>> AsVideoSessionConduit() = 0;
 
-  virtual webrtc::Call::Stats GetCallStats() const = 0;
+  virtual Maybe<webrtc::Call::Stats> GetCallStats() const = 0;
 
   NS_INLINE_DECL_THREADSAFE_REFCOUNTING(MediaSessionConduit)
 